faq/privacy-and-compliance.md

```diff
@@ -0,0 +1,41 @@
+---
+description: This article outlines the most frequently asked questions about privacy and compliance with SysKit Point.
+---
+
+# Privacy and Compliance FAQ
+
+## Where can I find SysKit Point Service License Agreement?
+
+Please visit (https://www.syskit.com/eula/) to review our Service License Agreement.
```

Comment:
u ovom članku su svi linkovi u zagradama. nešto ne valja sa sintaksom/formatom linkova